cmake/modules/FindBackends.cmake

```diff
@@ -0,0 +1,11 @@
+set(GLOW_BACKENDS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/lib/Backends")
```

Comment:
We have `GlowExternalBackends.cmake` already. Those ones should be taken into account as well. 

I feel like we need to build `GLOW_BACKENDS` variable that includes both in-tree and external backends. May be this logic should even live in the same cmake file.

Reply:
Hm... I kind of agree, but externalbackends/ has its own layout and logic.  I think this work is probably orthogonal (although perhaps if everything works out we can deprecate externalbackends and just use lib/Backends)